docs/release/release-tasks.md

```diff
@@ -406,7 +406,35 @@ We should inform at least the following providers via a new issue on their respe
  - Packet: https://github.com/kubernetes-sigs/cluster-api-provider-packet/issues/new
  - vSphere: https://github.com/kubernetes-sigs/cluster-api-provider-vsphere/issues/new
 
```

Comment:
Q: Should we start putting stuff like this in a separate file? We could have docs/release/communication/file and then just link it from here

Reply:
IMO it's fine to have here for now as a first step. This process should ideally be automated so it's not a load of copy-paste work though.

Reply:
Keeping it as is for now :)